patroni/postgresql.py

```diff
@@ -59,7 +60,8 @@ def __init__(self, config):
         self.callback = config.get('callbacks') or {}
         self.use_slots = config.get('use_slots', True)
         self._schedule_load_slots = self.use_slots
-        self._postgresql_conf = os.path.join(self._data_dir, 'postgresql.conf')
+        self._postgresql_conf = os.path.join(self._data_dir,
+                                             config.get('config', 'postgresql.conf'))
```

Comment:
hmmm, if we're renaming the files, then PipelineDB's suggested solution isn't going to work, because the new postgresql.conf won't get picked up.